pkg/utils/kubeconfig/kubeconfig.go

```diff
@@ -18,7 +18,12 @@ import (
 )
 
 // DefaultPath defines the default path
-var DefaultPath = clientcmd.RecommendedHomeFile
+var DefaultPath = func(env string) string {
```

Comment:
This might be a tiny bit clearer as a little helper function.

```golang
var DefaultPath = getDefaultPath()
```

Reply:
Aye, switching to `func` would be better for testing but means changing from _var_ to _func_.
If it looks good to maintainers, I can update my PR.

Reply:
Hi @prometherion, I like the suggestion to put the logic in a helper function as well, would you have time to update the PR? Thanks!

Reply:
sure @michaelbeaumont , just implemented with the latest commit!